server/index.js

```diff
@@ -7,10 +7,16 @@ const path = require('path');
 
 app.use(bodyParser.json());
 
-app.use(express.static(path.join(__dirname, '../public')))
+app.use('/', express.static('./public/'));
+app.use(/\/\d+\//, express.static('./public/'));
 
-app.get('/api/about', (req, res) => {
-  db.About.findOne()
+
+app.get('/api/about:id', (req, res) => {
```

Comment:
does this work? I'm used to seeing `'/api/about/:id'`; I don't know how `'api/about:id'` behaves